### DISCUSSION TITLE
Nowa instalacja - oczekuje na akceptacje

### DISCUSSION THREAD

**Discussion by Pablo010** [YetiForceCompany/YetiForceCRM#15874] [General]
Nowa instalacja - oczekuje na akceptacje.
Firma niezarejestrowana przez co część funkcjonalności nie działa, trwa to i trwa, jako można przyspieszyć?

**Comment by bpabiszczak** [comment]
użyj opcji sprawdź status.

**Reply by Pablo010** [reply]
Takie proste, dziękuję :-)
